examples/Orekit_example/orekit_integration.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
<blockquote>This example requires an <a href="https://gitlab.orekit.org/orekit-labs/python-wrapper/-/wikis/installation" target="_blank">orekit installation</a> via conda.</blockquote><p>This examples requires an <a href="https://gitlab.orekit.org/orekit-labs/python-wrapper/-/wikis/installation" target="_blank">orekit installation</a> via conda. To be used successfully, JAVA shall be installed.</p><p> </p><blockquote>Additional you will need to place the orekit <a href="https://gitlab.orekit.org/orekit/orekit-data" target="_blank">data</a> as orekit-data.zip in the same folder as this notebook.</blockquote><p>Additional you will need to place the orekit <a href="https://gitlab.orekit.org/orekit/orekit-data" target="_blank">data</a> as orekit-data.zip in the same folder as this notebook. N.B. The zip shall be not decompressed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>